guides/common/modules/proc_adding-custom-rpm-repositories-el.adoc

```diff
@@ -17,8 +17,10 @@ For more information, see xref:Creating_a_Custom_Product_{context}[].
 ** *Upstream URL*: `{yum_repo_baseos_url}`
 * *{yum_repo_appstream_name}*
 ** *Upstream URL*: `{yum_repo_appstream_url}`
+ifeval::["{os_name_anchor}" != "CentOS_Stream"]
```

Comment:
I'm unsure about this check. Is there any better way to solve this?

Reply:
I think this is fine as is. Alternatively, you could check if there's also an "extras" repo for CentOS Stream 9.

Reply:
I don't think there is an "extras" repository for CentOS Stream 9. At least I couldn't find any listed on https://mirror.stream.centos.org/9-stream/ and https://centos.pkgs.org/.

Reply:
There is:

````
$ podman run --rm -it centos:stream9

# dnf makecache
CentOS Stream 9 - BaseOS                                                                                                                                                      1.8 MB/s | 8.0 MB     00:04    
CentOS Stream 9 - AppStream                                                                                                                                                   2.2 MB/s |  19 MB     00:08    
CentOS Stream 9 - Extras packages                                                                                                                                              80 kB/s |  16 kB     00:00    
Last metadata expiration check: 0:00:01 ago on Wed Mar 20 15:07:50 2024.
Metadata cache created.
````

Reply:
Thanks to @maximiliankolb I found out that I was searching at the wrong place.
The "extras" repository for CentOS Stream 9 is located at https://mirror.stream.centos.org/SIGs/9-stream/.